core/trino-main/src/main/java/io/trino/execution/executor/TaskExecutor.java

```diff
@@ -90,6 +91,11 @@
     private final int minimumNumberOfDrivers;
     private final int guaranteedNumberOfDriversPerTask;
     private final int maximumNumberOfDriversPerTask;
+    private final DistributionStat allSplitsSizeDistribution = new DistributionStat();
```

Comment:
nit: you could move that to the bottom so that `final` variables are above non-final ones (code style)